android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Router.kt

```diff
@@ -97,6 +103,33 @@ abstract class Router<C : Parcelable, Permanent : C, Content : C, Overlay : C, V
         backStackFeature.accept(NewRoot(configuration))
     }
 
+    internal fun add(configurationKey: ConfigurationKey, configuration: Content) {
+        configurationFeature.accept(
+            Add(configurationKey, configuration)
+        )
+    }
+
+    internal fun remove(configurationKey: ConfigurationKey) {
+        configurationFeature.accept(
+            Remove(configurationKey)
+        )
+    }
+
+    internal fun activate(configurationKey: ConfigurationKey) {
+        configurationFeature.accept(
+            Activate(configurationKey)
+        )
+    }
+
+    internal fun deactivate(configurationKey: ConfigurationKey) {
+        configurationFeature.accept(
+            Deactivate(configurationKey)
+        )
+    }
+
+    internal fun getNodes(configurationKey: ConfigurationKey) =
+        (configurationFeature.state.pool[configurationKey] as? ConfigurationContext.Resolved<C>)?.nodes?.map { it.node }
```

Comment:
Report error if nothing found?

Reply:
Call site gets ```List<Node<*>>?``` as result, and has the responsibility to handle if it asked for something impossible.